notebooks/Primer/Primer.ipynb

```diff
@@ -2,10 +2,7 @@
  "cells": [
```

Comment:
Any instructions or anything for Exercise 2? Orphan cell at the bottom of the page.